src/data/repositories/UserD2ApiRepository.ts

```diff
@@ -39,6 +46,7 @@ export class UserD2ApiRepository implements UserRepository {
                 pageSize,
                 query: search !== "" ? search : undefined,
                 filter: otherFilters,
+                rootJunction: filters ? rootJunction : undefined,
```

Comment:
If you go to advanced filters, set mode=AND then mode=OR, the user list shows empty. The problem is here. This line correctly wants to skip `rootJuction` if no filters are enabled (as this breaks OR mode). But `filters` is an object, which in JS is always truthy. Also, even if it had properties, a filter is disabled when it has null/undefined values. Also, it should check `otherFilters`, which is the final value. A proposal:

```
const anyFilterEnabled = _(otherFilters).values().some()
```